react-app/src/components/toggle-switch/toggle-switch.css

```diff
@@ -0,0 +1,44 @@
+.toggle-switch {
+    height: 0;
+    width: 0;
+    visibility: hidden;
```

Comment:
NIT: This works, but in the future a simpler way to hide an element AND not get it to take up any space in the layout is to just use `display: none`. That'll remove it from the document entirely and you won't have to set a `height` and `width`.

Reply:
I always get visibility and display mixed up... thank you for the suggestion!

Reply:
Realistically I'm overanalyzing the demo-app but I've no headsets so its about the only thing I'm really good for when it comes to reviewing these right now lol.